pkg/operation/botanist/component/vpa/templates/crd-verticalpodautoscalers.tpl.yaml

```diff
@@ -22,99 +24,511 @@ spec:
     singular: verticalpodautoscaler
   scope: Namespaced
   versions:
-  - name: v1
+  - additionalPrinterColumns:
```

Comment:
When you diff this file with the upstream CRD there is a lot of no-op diff in (additional whitespace in the indentation). Can you check this and stick to the upsteam file? With such diff I would expect to see only the relevant modification (not no-op changes).

Reply:
Done.